### PR TITLE
Added mechanism for simple inclusion/exclusion of analyses

### PR DESCRIPTION
**Pull request by SundermannC**

### CODE REVIEW

**Comment by SundermannC** [comment]
@RSD6170 I guess this mechanism is not really fancy and we should improve this soon. For now, I just need something to turn of the SATZilla analyses.

File: src/main/java/org/collection/fm/FeatureStepAnalysis.java [hunk 1]
```diff
@@ -32,22 +27,35 @@ public static void main(String[] args) {
...
 
         analysis.initializeHandler(enumMap);
 
         List<File> files = FileUtils.getFileList(args[0]);
 
-        analysis.handleFiles(files, args.length == 1 ? "result.csv" : args[1], args.length == 1 ? "time.csv" : args[2], args.length == 1 ? "runstatus.csv" : args[3]);
+        analysis.handleFiles(files, args.length < 3 ? "result.csv" : args[1], args.length < 3 ? "time.csv" : args[2], args.length < 3 ? "runstatus.csv" : args[3]);
+    }
+
+    private static EnumMap<AnalysisStepsEnum, Integer> getEnumMapByCategory(String category, int timeout) {
```
Comment (RSD6170) [inline src/main/java/org/collection/fm/FeatureStepAnalysis.java]: Do we want some kind of error handling?

Comment (RSD6170) [inline src/main/java/org/collection/fm/FeatureStepAnalysis.java]: Move method to Enum-Class?

Comment (SundermannC) [reply src/main/java/org/collection/fm/FeatureStepAnalysis.java]: Done

Comment (SundermannC) [reply src/main/java/org/collection/fm/FeatureStepAnalysis.java]: Yes , that would be good, but I think we should replace the positional args with key-value args soon, as we already have multiple optional arguments.

Comment (RSD6170) [reply src/main/java/org/collection/fm/FeatureStepAnalysis.java]: A possible way could be [Apache CLI](https://commons.apache.org/proper/commons-cli/)

Comment (RSD6170) [reply src/main/java/org/collection/fm/FeatureStepAnalysis.java]: If you want, I could implement an exemplary version in a new branch

**Pull request merged**